app/src/main/java/com/smlnskgmail/jaman/hashchecker/components/hashcalculator/hash/CustomConstants.java

```diff
@@ -0,0 +1,12 @@
+package com.smlnskgmail.jaman.hashchecker.components.hashcalculator.hash;
+
+public class CustomConstants {
```

Comment:
Why we need `CustomConstants`? We already have `Constants` class? Or is it not for us?

Reply:
The Constants is in 
```java
  com.github.aelstad.keccakj.provider.Constants;
```
package. Therefore I created the CustomConstants class. Since we can not update com.github.aelstad.keccakj.provider.Constants

Reply:
Can you create a class with name like `HashConstants` and place to this file all hash types? Include MD-5 and other. Because we need to store all constants in one place.

Reply:
ok

Reply:
Hi @fartem I have make the changes can you verify